decidim-budgets/app/helpers/decidim/budgets/projects_helper.rb

```diff
@@ -127,10 +127,10 @@ def has_position?(project)
       end
 
       def filter_addition_type_values(added_count:)
-        [
-          ["all", { text: t("all", scope: "decidim.budgets.projects.project_filter"), count: nil }],
-          ["added", { text: t("added", scope: "decidim.budgets.projects.project_filter"), count: added_count }]
-        ]
+        filters = []
+        filters << ["all", { text: t("all", scope: "decidim.budgets.projects.project_filter"), count: nil }]
+        filters << ["added", { text: t("added", scope: "decidim.budgets.projects.project_filter"), count: added_count }] unless voting_finished?
```

Comment:
Two things:

1. I'd prefer to just remove this filter if we want to drop the "Added" filter, as just having a button that says "All" is not useful: 

![Screenshot of the "All" button](https://github.com/decidim/decidim/assets/717367/f97a6008-ed99-4096-98d5-9f8319fa4671)

2. Can you add a spec for this? I think a good place could be 
https://github.com/decidim/decidim/blob/b49c5e1398404e84baf55e9f9e8417468c03a8e4/decidim-budgets/spec/system/orders_spec.rb#L574

Reply:
Examples added